roles/setup_tftp/handlers/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Restart TFTP
+  ansible.builtin.service:
+    name: tftp.socket
+    state: restarted
+  become: true
+
+- name: Start TFTP
+  ansible.builtin.service:
+    name: tftp.socket
+    state: started
+    enabled: true
+  become: true
```

Comment:
Please move the `become` to the top of the block. This makes it much easier to determine which block have elevated privileges. 
```
- name: Restart TFTP
   become: true
   ansible.builtin.service
```